src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
 program
   .arguments('<address>')
   .action((address) => {
+    if (!address.length) {
+        throw new TypeError('Address is required');
     try {
       service.executeTimeEstimates(address).then(table => console.log(table));
     } catch (Error) {
       console.error('Could not get time estimates');
     }
-  })
-  .parse(process.argv);
+    service.executeTimeEstimates(address)
+        .then(table => console.log(table));
+} catch (Error) {
```

Comment:
Ah, no problem - that was just me mis-reading the merge changes. I switched it back in the latest commit after running through the tests.